src/iop/temperature.c

```diff
@@ -55,6 +55,8 @@ DT_MODULE_INTROSPECTION(3, dt_iop_temperature_params_t)
 #define DT_IOP_LOWEST_TINT 0.135
 #define DT_IOP_HIGHEST_TINT 2.326
 
+#define DT_COEFF_EPS 0.00001f
```

Comment:
Using equality on float is wrong anyway. We have plenty of them...

What about adding a feqf() routine in `common/math.h`? This could be reused in multiple places.

```
/* returns TRUE if a = b epsilon DT_COEFF_EPS */
gboolean feqf(float a, float b);
```

Reply:
maybe `gboolean feqf(float a, float, float eps)` ?